src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -476,4 +478,83 @@ public void Stop()
 
 		NativeMethods.Stop(IsAudio ? _htmlAudio.HtmlId : _htmlVideo.HtmlId);
 	}
+
+	public double _playbackRate;
+	public double PlaybackRate
+	{
+		get => _playbackRate;
+		set
+		{
+			_playbackRate = value;
+			if (IsVideo)
+			{
+				_htmlVideo.ExecuteJavascript("element.playbackRate = '" + _playbackRate.ToString("0.00", CultureInfo.InvariantCulture) + "';");
```

Comment:
@jeromelaban @ebariche Is this the right way?

Reply:
No